File: Introductory Problems/two knights/twoKnights.js
function twoKnights(n) {
  const array = Array.from({ length: n }, (_, i) => i + 1);
  array.forEach((k) => {
    const numOfSq = k * k;
    const comboOfValidSq = (numOfSq ** 2 - numOfSq) / 2;
    const inValidSq = (k - 1) * (k - 2) * 2 * 2;
    console.log(comboOfValidSq - inValidSq);
  });
}

twoKnights(8);

File: Introductory Problems/two knights/two_knights.py
def twoKnights(n): 
    array = [i+1 for i in range(n)]
    for k in array: 
        numOfSq = k*k
        comboOfValidSq = ((numOfSq)**2 - (numOfSq))/2
        inValidSq = (k-1)*(k-2)*2*2
        print(int(comboOfValidSq - inValidSq))

twoKnights(8)
